trex-container-app/server/scripts/trex-cfg-configure

```diff
@@ -31,7 +31,7 @@ port_obj_l2_template = {
     "src_mac": ""
 }
 
-CFG_FILE = "/etc/trex_cfg.yaml"
+CFG_FILE = "/usr/local/bin/example-cnf/trex_cfg.yaml"
```

Comment:
Here, just for my understanding, I'm curious why you decided to move the trex cfg file to `/usr/local/bin/example-cnf/` ?

Reply:
Because, if leaving it at `/etc/`, I would have needed to mount a volume on `/etc` folder to be able to create that file, and I didn't want to expose the whole `/etc` directory just for that